iocore/net/P_UnixNetVConnection.h

```diff
@@ -170,6 +170,38 @@ class UnixNetVConnection : public NetVConnection
   /////////////////////////////////////////////////////////////////
   UnixNetVConnection();
 
+  int
+  populate_protocol(char const **results, int n) const
+  {
+    int retval = 0;
+    if (n > 0) {
+      results[0] = options.get_proto_string();
+      retval++;
+      if (n > 1) {
+        results[1] = options.get_family_string();
+        retval++;
+      }
+    }
+    return retval;
+  }
+
+  const char *
+  protocol_contains(const char *tag) const
+  {
+    const char *retval   = NULL;
+    unsigned int tag_len = strlen(tag);
+    const char *test_tag = options.get_proto_string();
+    if (tag_len <= strlen(test_tag) && strncmp(tag, test_tag, tag_len) == 0) {
```

Comment:
Don't need to do the `strlen`, `strncmp` will stop (and fail) if the protocol tag has a null termination.